### ISSUE TITLE
Support for dark window frame

### ISSUE THREAD

**Issue by overheadhunter** [cryptomator/cryptomator-osx#5]
By adding a property to the [Info.plist](https://github.com/cryptomator/cryptomator-mac/blob/master/resources/app/Info.plist), we can support dynamic switching of the window frame:

```diff
  <key>NSHighResolutionCapable</key>
  <string>true</string>
+  <key>NSRequiresAquaSystemAppearance</key>
+  <false/>
  <key>NSUbiquitousContainers</key>
  <dict>
```

See also: [JDK-8214131](https://bugs.openjdk.java.net/browse/JDK-8214131).

This does not, of course, add a dark mode the the Cryptomator UI itself.

**Comment by overheadhunter** [comment]
At the moment we can not support this, because it breaks our context menus.

**Comment by tobihagemann** [comment]
Basically fixed since Cryptomator 1.5.0.

**Comment by overheadhunter** [comment]
> Basically fixed since Cryptomator 1.5.0.

Fixed with abdfc3e to be precise.
